### PR TITLE
Add Cambridge office to footer

### PR DESCRIPTION
**Pull request by Stormheg**
This PR adds the new Cambridge office to the footer of the website. https://torchbox.com/blog/cambridge/
Screenshot 2020-02-12 at 14 15 29

Mobile:
image

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -35,6 +35,21 @@ const Footer = ({ links, className }) => (
           </p>
         </li>
 
+        <li className={styles.footerAddress}>
+          <h2 className={styles.footerAddressTitle}>Historic Cambridge</h2>
+          <p>
+            Allia Future Business Centre
```
Comment (tomdyson) [inline src/components/footer/footer.js]: Can we drop 'Allia'? Sounds a bit cooler without it. :)

**Comment by NathHorrigan** [comment]
Yeah looks good to me :)

File: src/components/footer/footer.js (hunk 1 above)
Comment (Stormheg) [reply src/components/footer/footer.js]: Sure thing!

**Comment by tomdyson** [comment]
Thanks @Stormheg !

**Pull request merged**